modules/data.atmosphere/R/met.process.R

```diff
@@ -113,7 +177,34 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
 
         cmdFcn  = paste0(pkg,"::",fcn,"(",paste0("'",args,"'",collapse=","),")")
         new.files <- remote.execute.R(cmdFcn,host$name,user=NA, verbose=TRUE)
```

Comment:
new.files needs to be a list, otherwise the second iteration through this loop will blow away the first, and so on.